filters/contains-non-ascii/filter.py

```diff
@@ -0,0 +1,15 @@
+from interfaces.SentenceOperation import SentenceOperation
+from tasks.TaskTypes import TaskType
+
+
+class TextContainsNonAsciiFilter(SentenceOperation):
```

Comment:
Scope: I think this filter style of identifying types of characters could be extended to beyond only ASCII. For example, `containsEmoji`, `containsNonUtf8`, `containsCyrillicCharacters`, etc.

Reply:
Thanks for the feedback! Great idea!